### PR TITLE
3.0.0 alpha 2 preparation

### PR DESCRIPTION
**Pull request by Andy-Grigg**
Cherry-pick #149, #152, #153, #156

### CODE REVIEW

File: ansys-grantami-serverapi-openapi/pyproject.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ documentation = "https://grantami.docs.pyansys.com"
 readme = "README.md"
 keywords = ["Ansys", "OpenAPI"]
 classifiers = [
-    "Development Status :: 3 - Alpha",
+    "Development Status :: 5 - Production/Stable",
```
Comment (Andy-Grigg) [inline ansys-grantami-serverapi-openapi/pyproject.toml:17]: This will be reverted in a separate PR.

**Review by ludovicsteinbach — COMMENTED** [review]
What about #151? Or is #153 only included because it fixes a vulnerability and we generally don't cherry-pick dependency updates?

**Comment by Andy-Grigg** [comment]
@ludovicsteinbach That was my thought. Cherry-picking dependency updates is a pain because the lock file always conflicts, but I thought that was outweighed by addressing a vulnerability

**Review by ludovicsteinbach — APPROVED** [review]
